src/modules/rover_differential/module.yaml

```diff
@@ -120,3 +112,59 @@ parameters:
         increment: 0.01
         decimal: 2
         default: 0.5
+
+      RD_MAX_SPEED:
+        description:
+          short: Maximum speed
+          long: Maximum speed the rover can drive.
```

Comment:
Is this intended to be used as setpoint limit? Then I would specify it here.

Reply:
Specified it more in 817bd21